server/server/data_loader/arangoload.py

```diff
@@ -377,40 +377,39 @@ def generate_relationship_edges(change_tracker, relationship_dir, db):
     antispam = set()
     ll_edges = []
     for entry in tqdm(relationship_data):
-        remarks = entry.pop('remarks', None)
-        assert len(entry) == 1
         for r_type, uids in entry.items():
-            pass
-        full = [uid for uid in uids if not uid.startswith('~')]
-        partial = [uid for uid in uids if uid.startswith('~')]
-
-        # TODO: directionality has to be taken into account
-        # for some relationship kinds
-        for from_uid in full:
-            true_from_uids = get_true_uids(from_uid, all_uids)
-            if not true_from_uids:
-                print_once(f'Could not find any uids for: {from_uid}', antispam)
+            if r_type == 'retells':
+                r_type = 'retelling'
+            elif r_type == 'mentions':
+                r_type = 'mention'
+            elif r_type == 'parallels':
+                r_type = 'full'
+
+            from_uids = get_true_uids(uids[0], all_uids)
+            if not from_uids:
+                print_once(f'Could not find any uids for: {uids[0]}', antispam)
                 continue
-            for to_uids, is_partial in ((full, False), (partial, True)):
-                for to_uid in to_uids:
-                    if to_uid == from_uid:
-                        continue
-                    true_to_uids = get_true_uids(to_uid, all_uids)
-                    if not true_from_uids:
-                        if is_partial:
-                            print_once(f'Could not find any uids for: {to_uid}', antispam)
-                        continue
 
-                    for true_from_uid in true_from_uids:
-                        for true_to_uid in true_to_uids:
-                            ll_edges.append({
-                                '_from': true_from_uid,
-                                '_to': true_to_uid,
-                                'from': from_uid,
-                                'to': to_uid.lstrip('~'),
-                                'type': r_type,
-                                'partial': is_partial,
-                            })
+            for to_uid in uids[1:]:
+                if to_uid.startswith('~'):
+                    resembling = True
+                else:
+                    resembling = False
```

Comment:
1. Could we use `resembling = to_uid.startswith('~'):` ?
2. I'd propose naming boolean variables starting with `is_` and `are`. You decide.